maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt

```diff
@@ -15,12 +15,65 @@
  *
  */
 @file:Suppress("NOTHING_TO_INLINE")
+
 package com.google.maps.android.ktx
 
 import com.google.android.gms.maps.model.LatLng
 import com.google.maps.android.PolyUtil
 import com.google.maps.android.SphericalUtil
 
+/**
+ * Computes where the given [latLng] is contained on or near this Polyline within a specified
```

Comment:
`where the given [latLng] is contained on or near` -> `whether the given [latLng] lies on or near`

Reply:
should `this` be `[this]`? I'm not sure how `this` works with Kotlin docs.

Reply:
Just tested—`[this]` doesn't work in Kotlin docs